app/src/main/java/com/example/cinemapp/data/MovieRemoteDataSource.kt

```diff
@@ -13,6 +13,18 @@ interface MovieRemoteDataSource {
         @Query("page") page: Int = 1
     ): Response<MovieResponseDTO>
 
+    @GET("3/movie/popular")
```

Comment:
Could this number be moved to base url? Or just create a `const val` in `companion objet` so we can avoid "magic string" situation